R/data-importer-configuration.R

```diff
@@ -257,10 +264,6 @@ DataImporterConfiguration <- R6::R6Class(
       }
       super$initialize(ref)
       private$.dataImporterTask <- importerTask
-
-      private$.timeColumn <- rClr::clrCall(importerTask, "GetTime", ref)
```

Comment:
We don't cache the columns any more in the class, they are always retrieved from .NET object. Less error prone.